architect/systems/optical/spectrometers.py

```diff
@@ -303,14 +303,13 @@ def get_pointing_accuracy_constraint(
 
         return constraint_angle
 
-    def get_ground_target_error(self, orbital_altitude, skew_angle):
+    def get_ground_target_error(self, orbital_altitude, skew_angle, pointing_accuracy: unit.deg = 0):
```

Comment:
From here on out, all new functions will require type hints with units, like this:

```suggestion
    def get_ground_target_error(self, orbital_altitude: unit.m, skew_angle: unit.deg, pointing_accuracy: unit.deg = 0) -> unit.m:
```